frontend/src/components/ChatBox/ChatBox.tsx

```diff
@@ -235,7 +236,23 @@ function ChatBox({
 					</span>
 				</div>
 				<div className="control-buttons">
-					<Suspense fallback={<p>loading pdf library</p>}>
+					<Suspense
+						fallback={
+							<ThemedButton
+								className={'chat-button chat-button-disabled'}
+								onClick={() => {
+									console.error('This button is still loading');
```

Comment:
I'd be tempted to use a no-op for click, rather than put an error into the console.

`onClick={() => {}}`

However, that depends on what you do about my other comment for this fallback.

Reply:
well, the button shouldn't be clickable anyway, so yeah I'll make it an empty method